packages/ui-components/src/components/DataTable/editing/ParameterList.js

```diff
@@ -0,0 +1,167 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import PropTypes from 'prop-types';
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
+import { Divider as BaseDivider } from '@material-ui/core';
+import { Add as AddIcon } from '@material-ui/icons';
+import Grid from '@material-ui/core/Grid';
+import BaseDeleteOutlinedIcon from '@material-ui/icons/DeleteOutlined';
+import { Checkbox as BaseCheckbox, Select, TextField } from '../../Inputs';
+import { IconButton as BaseIconButton } from '../../IconButton';
+import { FlexStart } from '../../Layout';
+import { ParametersType } from './types';
+import { FilterTypeOptions } from '../PreviewFilters';
+import { Button } from '../../Button';
+
+const ListDiv = styled.div`
+  overflow-x: hidden;
+  overflow-y: auto;
```

Comment:
Better to have the modal grow vertically if the list gets too long. This scrolling section inside a scrolling section is really awkward UX imo.

Reply:
Good point 👍. Added.